### PR TITLE
new text file

### PR DESCRIPTION
**Pull request by nadimalabbas**
new test file body

### CODE REVIEW

**Comment by nadtesttest** [comment]
yes

**Comment by nadimalabbas** [comment]
no

**Comment by nadtesttest** [comment]
can i trigger it

**Comment by nadimalabbas** [comment]
comment from me

**Comment by nadimalabbas** [comment]
yes from me again

**Comment by nadtesttest** [comment]
again

**Comment by nadtesttest** [comment]
123

**Comment by nadtesttest** [comment]
a new chain of comments

**Comment by nadtesttest** [comment]
1234

**Comment by nadtesttest** [comment]
123123

**Comment by nadimalabbas** [comment]
123

**Comment by nadtesttest** [comment]
123

**Comment by nadtesttest** [comment]
123

File: new.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+this is new file
+this is 2nd line
```
Comment (nadtesttest) [inline new.txt:2]: I meant you should add a comma

File: new.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+this is new file
```
Comment (nadtesttest) [inline new.txt:1]: please add a comma here

**Review by nadtesttest — COMMENTED** [review]
this is a comment review

**Review by nadtesttest — CHANGES_REQUESTED** [review]
I am requesting changes

**Review by nadtesttest — CHANGES_REQUESTED** [review]
requesting something

**Review by nadtesttest — CHANGES_REQUESTED** [review]
I request these changes

**Review by nadtesttest — CHANGES_REQUESTED** [review]
request changes again

**Pull request closed without merging**